### ISSUE TITLE
Show transaction input and inputs in details. [Feature]

### ISSUE THREAD

**Issue by Transisto** [BlueWallet/BlueWallet#3099]
I'm still experiencing weird coin selection behavior with Blue.

I'd like to be able to see what transaction the wallet is about to do before sending it.

Currently copy pasting the raw transaction in electrum but would be great to have it decoded in the wallet.

**Comment by ncoelho** [comment]
Are you using a custom wallet type, or what is your derivation path?

**Comment by Transisto** [comment]
Using a Blue wallet generated seed, Can only see the raw transaction + the TO address fee ect, Would like to see the inputs selected.

**Comment by Overtorment** [comment]
you can instead of broadcasting open transaction details screen, and there will be an option to decode hex on coinb.in

**Comment by Transisto** [comment]
> you can instead of broadcasting open transaction details screen, and there will be an option to decode hex on coinb.in

Good idea, Thanks, still shouldn't be that hard to implement locally since every previous transaction show these details well laid out.

**Comment by Overtorment** [comment]
its not hard, but this is exactly the spot where you want an independent 3rd party to verify your transaction prior to broadcast

**Comment by Transisto** [comment]
hmm why a 3rd party? to double check the balance of the inputs?
